subsys/power/device_pm.c

```diff
@@ -8,6 +8,7 @@
 #include <kernel.h>
 #include <device.h>
 #include <sys/__assert.h>
+#include <sys/check.h>
```

Comment:
why is this needed?

Reply:
leftover from a check I had added but end up removing.